Cargo.toml

```
@@ -11,10 +11,15 @@ keywords = ["battery", "linux", "macos", "windows", "freebsd"]
 license = "ISC"
 build = "build.rs"
 
+[features]
+config-schema = ["schemars"]
+
 [dependencies]
 cfg-if = "1.0"
 num-traits = { version = "0.2", default_features = false }
 uom = { version = "0.30", features = ["autoconvert", "f32", "si"] }
+serde = { version = "1.0.136", features = ["derive"] }
```

Starship isn't the only consumer of this library any more, because wezterm and bottom just started using this crate, so `serde` should be optional too:
```suggestion
serde = { version = "1.0.136", features = ["derive"], optional = true }
```

I am still in favour of removing `__Nonexhaustive` though.